client-side/src/components/MyButton.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import './MyButton.scss';
+
+import PropTypes from 'prop-types';
+
+import Button from '@mui/material/Button';
+
+
+const MyButton = ({className, label, onClick, size, disabled = false})=>{
+    return(
+        <Button 
+        className={['myButton', `myButton-${className}`].join(' ')}
+        onClick={onClick}
+        size={size}
+        disabled={disabled}
+        >
+        {label}
+        </Button>
+    );
+};
+
+MyButton.propTypes = {
+    
+    className: PropTypes.string,
+    
+    //How large should the button be?
+    size: PropTypes.oneOf(['small', 'medium','large']).isRequired,
+    
+    //Button contents
+    label: PropTypes.string.isRequired,
+    
+    //Click handler
+    onClick: PropTypes.func.isRequired,
+
+    //Optional is disabled
+    disabled: PropTypes.bool,
```

Comment:
you should add defaultProps for all props that are not required.
it will be something like:
MyButton.defaultProps: {
  className: 'primary',
  disabled: false
}
I don't know if I wrote the correct default values, but this is the structure

Reply:
I did it at first but I had this worning: **GenericButton: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.** so i changed it...

Reply:
do you want me to change it again?
and don't pay attention to this worning?

Reply:
ok, if so you are correct, I didn't know about this warning.
great that you saw this. leave it with the default parameters.